pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -101,6 +101,10 @@ stdenv.mkDerivation {
     ./patches/2.31/0001-x86-Properly-add-X86_ISA_1_NEEDED-property.patch
   ]
   ++ lib.optional stdenv.targetPlatform.isiOS ./support-ios.patch
+  ++ # This patch was suggested by Nick Clifton to fix
+     # https://sourceware.org/bugzilla/show_bug.cgi?id=16177
+     # It can be removed when that 7-year-old bug is closed.
```

Comment:
Please add to the comment what the patch's _purpose_ is / why nixpkgs needs it; it's not immediately obvious even from the issue description. Linking to https://github.com/NixOS/nixpkgs/issues/85924 would also be fine.